pontoon/sync/vcs_models.py

```diff
@@ -134,7 +135,7 @@ def resources_for_path(self, path):
                 filenames = [f for f in filenames if not f.endswith('region.properties')]
 
```

Comment:
@mathjazz done :)

On Tue, Dec 22, 2015 at 12:23 PM, Matjaž Horvat <notifications@github.com>
wrote:

> In pontoon/sync/vcs_models.py
> <https://github.com/mozilla/pontoon/pull/312#discussion_r48244860>:
>
> > @@ -134,7 +135,7 @@ def resources_for_path(self, path):
> >                  filenames = [f for f in filenames if not f.endswith('region.properties')]
> >
>
> It would be more optimal to skip hidden folders here entirely.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/pontoon/pull/312/files#r48244860>.
>



-- 
Kind regards

​Jarek "jotes" Śmiejczak,
​P​
ython programmer​ by day, mozillian by heart..
Homepage: http://joyfulhack.in, Github: http://github.com/jotes
Mobile: +48693027040, Mozillian: http://mozillians.org/u/jotes